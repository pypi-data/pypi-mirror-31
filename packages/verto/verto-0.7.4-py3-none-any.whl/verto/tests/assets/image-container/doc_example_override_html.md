{image file-path="http://placehold.it/350x150" caption="true" source="https://placehold.it/"}

This is the caption text.

{image end}
