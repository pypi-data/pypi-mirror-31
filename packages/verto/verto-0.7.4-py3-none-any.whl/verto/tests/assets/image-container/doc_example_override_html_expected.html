<div class="text-center">
<img class="rounded img-thumbnail" src="http://placehold.it/350x150" />
<h4>This is the caption text.</h4>
</div>
